recipes/emacs/build.sh

```diff
@@ -0,0 +1,7 @@
+if [ "$(uname)" == "Darwin" ]; then
+    ./configure  --prefix=$PREFIX --without-x
+else
+    ./configure  --prefix=$PREFIX --x-includes=$PREFIX/include --x-libraries=$PREFIX/lib
+fi
+
+make -j4  && make -j4  install
```

Comment:
Yeah, for now that's best.  I think Circle CI and AppVeyor offer 2 CPUs, but not Travis.